ppml/trusted-dl-serving/base/Dockerfile

```diff
@@ -0,0 +1,79 @@
+ARG BIGDL_VERSION=2.3.0-SNAPSHOT
+ARG TINI_VERSION=v0.18.0
+ARG BASE_IMAGE_NAME
+ARG BASE_IMAGE_TAG
+ARG JDK_VERSION=11
+
+#Stage.1 Torchserve Frontend
+FROM $BASE_IMAGE_NAME:$BASE_IMAGE_TAG as temp
+ARG http_proxy
+ARG https_proxy
+ARG JDK_VERSION
+ENV JDK_HOME                /opt/jdk${JDK_VERSION}
+ENV JAVA_HOME               /opt/jdk${JDK_VERSION}
+
+RUN apt-get install -y openjdk-${JDK_VERSION}-jdk && \
+    mkdir -p ${JAVA_HOME} && \
+    cp -r /usr/lib/jvm/java-${JDK_VERSION}-openjdk-amd64/* ${JAVA_HOME} && \
+    git clone https://github.com/analytics-zoo/pytorch-serve.git && \
+    cd pytorch-serve/frontend && \
+    ./gradlew clean assemble && \
+    mkdir -p /ppml/torchserve && \
+    mv server/build/libs/server-1.0.jar /ppml/torchserve/frontend.jar
+
+FROM $BASE_IMAGE_NAME:$BASE_IMAGE_TAG
+ARG http_proxy
+ARG https_proxy
+ARG no_proxy
+ARG TINI_VERSION
+ENV TINI_VERSION                        $TINI_VERSION
+ARG JDK_VERSION
+ENV JDK_HOME                            /opt/jdk${JDK_VERSION}
+ENV JAVA_HOME                           /opt/jdk${JDK_VERSION}
+# Environment used for build pytorch
+RUN mkdir /ppml/examples && \
+    mkdir /ppml/torchserve
+
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /sbin/tini
+ADD ./entrypoint.sh /opt/entrypoint.sh
```

Comment:
Shall we need an **entrypoint** for this image?
If yes, what should be included inside the entrypoint?